### ISSUE TITLE
Recurrent Neural Network Spam Detection

### ISSUE THREAD

**Issue by mderosa001** [mderosa001/Capstone_Issues#17]
### Title
Spam Detection using Recurrent Neural Networks
### URL
https://www.researchgate.net/publication/341426514_Spam_Detection_using_Recurrent_Neural_Networks
### Summary 
Traditional machine learning approaches for spam detection suffer from performance degradation with large datasets. To address this, the authors propose a spam detection model using recurrent neural networks, specifically the BiGRU (Bidirectional Gated Recurrent Unit) model, which achieves an accuracy of 99.07%. The paper includes a literature review of various spam detection techniques, including neural networks, support vector machines (SVM), and feature selection methods. It also explores hyperparameter tuning and performance evaluation metrics.
### Key Points 
Neural Networks are the key for higher accuracy scores
Hyperparameter tunning 

### Citation
@article{article,
author = {Hariprasad, Karishma and Gopu, Akila and Govindasamy, V.},
year = {2020},
month = {05},
pages = {313-318},
title = {Spam Detection using Recurrent Neural Networks},
volume = {6},
journal = {International Journal for Research in Engineering Application & Management},
doi = {10.35291/2454-9150.2020.0305}
}
